client/src/services/CosmosFetchService.ts

```diff
@@ -16,40 +16,54 @@ const COSMOS_API_KEY = process.env.COSMOS_API_KEY;
 
 /// /////////////////////////////////////////////
 
-const COSMOS_API_URL = 'https://xdd.wisc.edu/sets/xdd-covid-19/cosmos/api/search';
+const COSMOS_API_URL_PREFIX = process.env.NODE_ENV === 'development'
+  ? 'https://xdd.wisc.edu'
```

Comment:
While we are setting up DONU and S3 endpoint in the `.env` we should add COSMOS and EMMAA as well.

Reply:
We are passing COSMOS and S3 through a proxy as they contain API keys that we do not want to expose. EMMAA and Donu are public APIs so there is no reason to call them via a proxy.

Reply:
Yes, what I meant is just to store the URL inside a environment variable like DONU and S3.